### PR TITLE
Upgrade Slog to 2.5.1

### PR DESCRIPTION
**Pull request by CodeSandwich**
The goal is to clean up some ugliness around handling `slog::FilterLevel`

### CODE REVIEW

File: jormungandr/Cargo.toml [hunk 1]
```diff
@@ -36,6 +36,7 @@ cryptoxide = "0.1"
 futures    = "0.1"
 http = "0.1.16"
 hyper = "0.12"
+lazy_static = "1.3"
```
Comment (NicolasDP) [inline jormungandr/Cargo.toml]: why did you move this line?

Comment (NicolasDP) [reply jormungandr/Cargo.toml]: my bad...

**Comment by rinor** [comment]
slog [2.5.1](https://crates.io/crates/slog/2.5.1) has been **yanked** and is causing issues with jormunandr v0.3.1 source install. Can we use 2.5.0 or https://github.com/slog-rs/slog/commit/c8aa6770fd160330d032511ce1258f093d2d9df3.

**Comment by CodeSandwich** [comment]
I've asked what's wrong with 2.5.1: https://github.com/slog-rs/slog/issues/219, when we know that, it will be easier to decide what to do next.
